include/modem/at_monitor.h

```diff
@@ -45,6 +45,18 @@ struct at_monitor_entry {
 	bool paused;
 };
 
+/**
+ * @brief AT monitor entry (ISR).
+ */
+struct at_monitor_isr_entry {
```

Comment:
Strictly speaking we wouldn't, but the linker section is named after the type. If we used the same type, all monitors would end up in the same section and 1) we would need to add a flag to the `struct`, for example `dispatch_in_isr` and 2) we would need to loop all the monitors in the ISR, instead of just the ISR monitors to check for the flag above.

Reply:
Ok, thank you for clarifying, nevermind then.